.github/workflows/ci.bazelrc

```diff
@@ -0,0 +1,20 @@
+# This file contains Bazel settings to apply on CI only.
+# It is referenced with a --bazelrc option in the call to bazel in ci.yaml
+common --curses=no
+common --enable_bzlmod
```

Comment:
This config is only used by `bazel-contrib/.github/.github/workflows/release_ruleset.yaml@v4`